medicines/web/src/services/search-query-normalizer.ts

```diff
@@ -10,7 +10,7 @@ const escapeSpecialWords = (word: string): string =>
   word.replace(/(\|\||&&|\bAND\b|\bOR\b|\bNOT\b)/gi, `\\$1`);
 
 const preferExactMatchButSupportFuzzyMatch = (word: string): string =>
-  `${word}~${searchWordFuzziness} ${word}^${searchExactnessBoost}`;
+  `(${word}~${searchWordFuzziness}||${word}^${searchExactnessBoost})`;
```

Comment:
You need the space in between the `||` operator and the words. Also, just a design note as it doesn't seem to affect behaviour: can we put _exact_ first, _fuzzy_ second?